### PR TITLE
tools/ops - org-pr-monitor - fix output refactor move of MetricsOutput to c7n.aws module

### PR DESCRIPTION
**Pull request by thisisshi**
After the refactor to c7n.output, the import was moved to c7n.resources.aws-- updated to reflect that and fix up the subclass

edit: there's a drive by on adding GetInventoryConfiguration to the list of s3 denied methods as well

### CODE REVIEW

File: c7n/resources/aws.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ class MetricsOutput(Metrics):
     permissions = ("cloudWatch:PutMetricData",)
     retry = staticmethod(utils.get_retry(('Throttling',)))
 
-    def __init__(self, ctx, config=None):
+    def __init__(self, ctx, config={}):
```
Comment (kapilt) [inline c7n/resources/aws.py]: mutables in default args are always suspect

File: tools/ops/org-pr-monitor.py [hunk 2]
```diff
@@ -89,6 +90,12 @@ def _format_metric(self, key, value, unit, dimensions):
             d['Dimensions'].append({"Name": k, "Value": v})
         return d
 
+    def get_timestamp(self):
```
Comment (kapilt) [inline tools/ops/org-pr-monitor.py]: given no compatibility concern for this one, let's just go to utc unconditionally.

File: c7n/resources/aws.py (hunk 1 above)
Comment (thisisshi) [reply c7n/resources/aws.py]: yup, reverted

File: tools/ops/org-pr-monitor.py (hunk 2 above)
Comment (thisisshi) [reply tools/ops/org-pr-monitor.py]: sounds good

**Pull request merged**